src/Compilers/CSharp/Test/Symbol/Symbols/StaticAbstractMembersInInterfacesTests.cs

```diff
@@ -29953,13 +29953,16 @@ .locals init (int V_0)
         }
 
         [Theory]
-        [CombinatorialData]
-        public void ConsumeAbstractConversionOperator_11([CombinatorialValues("implicit", "explicit")] string op)
+        [InlineData("implicit", false)]
+        [InlineData("implicit", true)]
+        [InlineData("explicit", false)]
+        [InlineData("explicit", true)]
+        public void ConsumeAbstractConversionOperator_11([CombinatorialValues("implicit", "explicit")] string op, bool needCast)
```

Comment:
Can we remove `CombinatorialValues`?
nit: `useCast` would make more sense than `needCast`